packages/react/src/components/Forms/Select/mono-searchable.tsx

```diff
@@ -51,6 +86,27 @@ export const SelectMonoSearchable = forwardRef<SelectHandle, SubProps>(
       downshiftReturn.inputValue,
     ]);
 
+    useEffect(() => {
+      if (
+        typeof props.defaultValue === "string" &&
+        typeof props.options === "function"
+      ) {
+        (async () => {
+          await computeDefaultOptionToSelect(String(props.defaultValue));
+        })();
+      }
```

Comment:
To improve code maintainability and ease this method you could write something like that I think :

```suggestion
      if (
        props.defaultValue !== undefined &&
        typeof props.options === "function"
      ) {
            computeDefaultOptionToSelect(props.defaultValue);
      }
```